src/pat/tooltip/tooltip.js

```diff
@@ -26,7 +26,7 @@ define([
     parser.addArgument("height", "auto", ["auto", "max"]);
     parser.addArgument("trigger", "click", ["click", "hover"]);
     parser.addArgument("closing", "auto", ["auto", "sticky", "close-button"]);
-    parser.addArgument("source", "title", ["auto", "ajax", "content", "title"]);
+    parser.addArgument("source", "title", ["auto", "ajax", "content_html", "title"]);
```

Comment:
Yes, Alexander told me this afternoon that with just content the contents get placed in a <p> tag. I may assume that flow elements such as strong and em and also links may still work then, as they are allowed in a p-tag, right? So for now, since we only want this link, the current content method should be sufficient. Or does it escape the HTML code and thus printing them visible to the user?

> Op 10 sep. 2015, om 20:13 heeft Rok Garbas <notifications@github.com> het volgende geschreven:
> 
> In src/pat/tooltip/tooltip.js:
> 
> > @@ -26,7 +26,7 @@ define([
> >      parser.addArgument("height", "auto", ["auto", "max"]);
> >      parser.addArgument("trigger", "click", ["click", "hover"]);
> >      parser.addArgument("closing", "auto", ["auto", "sticky", "close-button"]);
> > -    parser.addArgument("source", "title", ["auto", "ajax", "content", "title"]);
> > +    parser.addArgument("source", "title", ["auto", "ajax", "content_html", "title"]);
> 
> @pilz good catch. i must have changed that like like 10 times before pushing.
> @cornae currently if text is passed as we will insert that text into
> 
> element ... now with content-html we insert this in
> 
> element.
> 
> i do agree content option should be also handling html as well as text, but you know where you've been using it and how much can go wrong.
> 
> safe option until then would be to add content-html option
> 
> —
> Reply to this email directly or view it on GitHub.
>